### PR TITLE
Add minutes from workshop and debrief meeting

### PR DESCRIPTION
**Pull request by lwjohnst86**
Please everyone review and approve this to make sure nothing was missed or other things we should add to it. :D

### CODE REVIEW

File: events/2023-10-in-person/index.qmd [hunk 1]
```diff
@@ -80,4 +80,126 @@ some topics and less on others.
...
+    - They had good process/flow for guiding us, they kept good pace and kept on time
+    - They were very professional and friendly to work with, good seniority/experience for them
+    - Obvious they had read up on what we were doing and were prepared
+    - Helped us to focus down on what things would be version 1 vs later versions, as well as identifying hotspots and things to move on with
+    - It was really useful to see how they work as well, though maybe not loving the purely scrum style they use
+- Hiring them and what/how they can work with us
+    - Could either hire them after we have a core functional box or hire them to help use develop our process and workflow (getting started, project management, teamwork practices, "how to work" rather than "what to work on")
+    - If hired for process, we'll need to give them clear requirements for what we want from them
+        - We might need some time to sit down and define the requirements for designing the how to work together
+    - [X] Action: To contact them about hiring them, what are the next steps? How them charge
```
Comment (signekb) [inline events/2023-10-in-person/index.qmd:113]: them -> they

**Review by K-Beicher — APPROVED** [review]
Can't of the top of my head see anything missing.

File: events/2023-10-in-person/index.qmd [hunk 2]
```diff
@@ -80,4 +80,126 @@ some topics and less on others.
...
+    - [ ] Action: Need to spend some time writing doc on how to work together, the process. This could be the next “focus stage”, on starting to write out a design document
+    - [ ] Action: Make a detailed list of what outputs we would like from them
+        - E.g. training material or documentation on working together and their process
+        - E.g. finalize the design doc (maybe another review of it)?
+- Things we could take from them
+    - Their approach to visualizing things (e.g. for instance with the vertical lines)
+        - For virtual settings: Look into online tools that help with doing the visualize
```
Comment (signekb) [inline events/2023-10-in-person/index.qmd:120]: Thought: They said that they would use Figma to digitalise the event storming boards. Maybe we could as well? Figma allows for real-time collaboration (not sure whether it's a free feature though): https://www.figma.com/collaboration/

File: events/2023-10-in-person/index.qmd [hunk 3]
```diff
@@ -80,4 +80,126 @@ some topics and less on others.
...
+        - [ ] Action: Richard make the first draft of template
+    - Make a PR template
+        - That includes a checklist of things to do
+        - Including “has unit test been added”
+    - Every update meeting comes with commit
+    - Nothing pushed to main, it has to go through PR and review
+    - If one of us are assigned or asked for review, work on it
+    - Always have someone assigned to everything
+    - Not working on side projects
+    - Annelli will attend an update meeting on
```
Comment (signekb) [inline events/2023-10-in-person/index.qmd:162]: Tuesday? And once per month?

File: events/2023-10-in-person/index.qmd [hunk 4]
```diff
@@ -80,4 +80,126 @@ some topics and less on others.
...
+    - Building monthly sprints, with a period of roadmapping in those days
+    - Use Kanban board with constrained time periods to use for the focus phases, with dedicated time pre-phase to describe and plan together what needs to be done and how long each task takes
+    - Focus on smaller focused periods of working, with iterative development/feedback and smaller units of roadmapping.
+    - [ ] Action: Schedule time shortly 
+    - Have twice-weekly update meetings
+    - Limit external meeting
+    - Make use of Issue’s
+        - Issue templates for how to structure or fill in Issue/task
+        - [ ] Action: Richard make the first draft of template
+    - Make a PR template
```
Comment (signekb) [inline events/2023-10-in-person/index.qmd:154]: Make this an action?

File: events/2023-10-in-person/index.qmd [hunk 5]
```diff
@@ -80,4 +80,126 @@ some topics and less on others.
...
+    - Annelli will attend an update meeting on
+- Tasks to do next
+    - Converting over the digital copies and updating documentation
+        - [ ] Action: Signe and Kris will convert
+    - Focusing on diagrams and visualizations in design docs, and updating the existing ones and adding more ones
+    - Have more definitions and lists on the definitions of terms
+    - Build up Actions/CI
+    - Pre-commits setup
+    - Basic infrastructure for working together
+    - Make issue template that includes a very detailed requirements for functionality (including input and output of functions/code)
```
Comment (signekb) [inline events/2023-10-in-person/index.qmd:171]: Actions as well? :)

File: events/2023-10-in-person/index.qmd [hunk 6]
```diff
@@ -80,4 +80,126 @@ some topics and less on others.
...
+- Moving forward, tracking progress, and building trust
+    - Building monthly sprints, with a period of roadmapping in those days
+    - Use Kanban board with constrained time periods to use for the focus phases, with dedicated time pre-phase to describe and plan together what needs to be done and how long each task takes
+    - Focus on smaller focused periods of working, with iterative development/feedback and smaller units of roadmapping.
+    - [ ] Action: Schedule time shortly 
+    - Have twice-weekly update meetings
+    - Limit external meeting
+    - Make use of Issue’s
+        - Issue templates for how to structure or fill in Issue/task
+        - [ ] Action: Richard make the first draft of template
```
Comment (signekb) [inline events/2023-10-in-person/index.qmd:153]: Oh! I have created a first draft in the .github repo for a user story issue template

File: events/2023-10-in-person/index.qmd [hunk 7]
```diff
@@ -80,4 +80,126 @@ some topics and less on others.
...
+    - Pre-commits setup
+    - Basic infrastructure for working together
+    - Make issue template that includes a very detailed requirements for functionality (including input and output of functions/code)
+    - Review process?
+        - PR
+- Misc items
+    - Ask Mjolner SQL injection (and looking into things)
+    - Frictionless data, pull it in?
+    - Workflow, each project/package/app would have its own design doc and diagrams, etc?
+- Requirements for how to work together effectively as a team
```
Comment (signekb) [inline events/2023-10-in-person/index.qmd:178]: We also talked about best workflow practices such as aiming for multiple commits per day and pushing our work at the end of the day

**Review by signekb — APPROVED** [review]
Looks good! I have added a couple of comments on items that could be actions as well as a couple of clarifications. I'm looking forward to the blogpost on the workshop! :)

File: events/2023-10-in-person/index.qmd [hunk 8]
```diff
@@ -80,4 +80,126 @@ some topics and less on others.
...
+    - Obvious they had read up on what we were doing and were prepared
+    - Helped us to focus down on what things would be version 1 vs later versions, as well as identifying hotspots and things to move on with
+    - It was really useful to see how they work as well, though maybe not loving the purely scrum style they use
+- Hiring them and what/how they can work with us
+    - Could either hire them after we have a core functional box or hire them to help use develop our process and workflow (getting started, project management, teamwork practices, "how to work" rather than "what to work on")
+    - If hired for process, we'll need to give them clear requirements for what we want from them
+        - We might need some time to sit down and define the requirements for designing the how to work together
+    - [X] Action: To contact them about hiring them, what are the next steps? How them charge
+    - [ ] Action: Need to spend some time writing doc on how to work together, the process. This could be the next “focus stage”, on starting to write out a design document
+    - [ ] Action: Make a detailed list of what outputs we would like from them
```
Comment (rqding) [inline events/2023-10-in-person/index.qmd:115]: I think we also should decide which module we will work on first

**Pull request merged**